string input_text
---
string output_text
